utils/fasm/test/test_fasm.cpp

```diff
@@ -99,17 +110,17 @@ TEST_CASE("fasm_integration_test", "[fasm]") {
             continue;
         }
 
-        if(!found_mux1 && line == "CLB.FLE9.OUT_MUX.LUT") {
+        if(!found_mux1 && line.find(".OUT_MUX.LUT") != std::string::npos) {
```

Comment:
Can we make this precise again? E.g. maybe examine the placement and predict what the prefix is?